src/Provider/Pelias/Pelias.php

```diff
@@ -166,7 +166,12 @@ protected function executeQuery(string $url): AddressCollection
                     'east' => $location['bbox'][0],
                 ];
             } else {
-                $bounds = null;
+                $bounds = [
```

Comment:
maybe worth adding a test case to cover this bug?

Reply:
Good idea! 👍

Reply:
Test added but I don't have an API key to add the cached response (I requested one).